calculator_simple/src/app_window.hpp

```diff
@@ -0,0 +1,102 @@
+/**
+ * @file
+ * @brief   (Declarations) View Layer of a simple Calculator application.
+ * 
+ *
+ * @author     Tommy Vegetables <leafyrabbet@gmail.com>
+ * @date       2020-05-17
+ * @copyright  2020, TenonGarden Productions
+ */
+#ifndef HPP_APP_WINDOW_HPP
+#define HPP_APP_WINDOW_HPP
+
+#include <glibmm/ustring.h>
+
+#include <sigc++/sigc++.h>
+
+#include <gtkmm/window.h>
+#include <gtkmm/box.h>
+#include <gtkmm/drawingarea.h>
+#include <gtkmm/grid.h>
+#include <gtkmm/label.h>
+#include <gtkmm/adjustment.h>
+#include <gtkmm/scale.h>
+#include <gtkmm/button.h>
+
+#include <array>
+
+#include "logic_calculator.hpp"
+
+
+class AppWindow :
+              public Gtk::Window
+{
+   public:
+      AppWindow();
+      virtual ~AppWindow();
+
+      void handle_display_update();
+      bool handle_radix_change(const Gtk::ScrollType & scroll_type, const double & radix_new);
+      void handle_btn_0();
+      void handle_btn_1();
+      void handle_btn_2();
+      void handle_btn_3();
+      void handle_btn_4();
+      void handle_btn_5();
+      void handle_btn_6();
+      void handle_btn_8();
+      void handle_btn_7();
+      void handle_btn_9();
+      void handle_btn_a();
+      void handle_btn_b();
+      void handle_btn_c();
+      void handle_btn_d();
+      void handle_btn_e();
+      void handle_btn_f();
+ 
+   protected:
+ 
```

Comment:
I believe most of the `public` member-functions can be moved here under the `protected` access-attribute or under `private`, without issue. They were moved to `public` during development when there was a misread compiler error due to improper includes for `gcc-9`, at the time.

Reply:
Resolved in 11c71e4